db.json

```diff
@@ -4,22 +4,26 @@
       {
         "id": 1,
         "name": "architecture",
-        "description": "Some archy description."
+        "description": "Some archy description.",
+        "tooltip": "{proportion} of users have the {x} architecture"
```

Comment:
> Aren't template strings a built in? I just googled "Javascript template strings" and there's something that looks like ${proportion} of users have ${x} architecture and wrapped in backticks?

Sort of. But it's not so much about string replacement, like `sprintf` and the like, so much as it is about string interpolation. For example:

```javascript
const name = 'Tom';
const age = 30;

console.log(`${name} is ${age} years old.`);
```

> I'm also curious if the metric store can tap into the metric metadata info? I'll defer to @openjck on how best to structure that. Does it make sense to push some of this down to the metric API?

Do you mean outputting that tooltip summary directly from */metrics/*? I would be fine with that. At the same time, since it constitutes a presentation of raw data, I think it would be appropriate for clients to do that. Not all client applications are going to want to present that data the same way.